.github/workflows/test-integration.yml

```diff
@@ -0,0 +1,52 @@
+name: test-integration
+on:
+  pull_request:
+  push:
+#    branches: [ master, develop ]
```

Comment:
Dev leftover?

Reply:
Missed that one, It has been uncommented